test/e2e/components/vl-select.js

```diff
@@ -82,9 +76,16 @@ class VlSelect extends VlElement {
 
     async _clickOption(option) {
         if ((await this.isDressed())) {
-            await this._openDressedDropdown();
+            await this.open();
         }
-        return option.webElement.click();
+        return option.click();
+    }
+
+    async open() {
+        if (await this._isOpen()) {
+            return Promise.resolve();
+        }
+        return (await this._getDressedContainer()).click();
```

Comment:
Vind het iets makkelijker lezen als we niet expliciet `Promise.resolve()` moeten aanroepen. Je kan er ook van maken:

```suggestion
        const isOpen = await this._isOpen();
        if (!isOpen) {
            const container = await this._getDressedContainer();
            container.click();
        }
```